tests/check_included_tables.sh

```diff
@@ -18,14 +18,17 @@ has_metadata() {
     return 1
 }
 
-for table in $(dirname $0)/../tables/*; do
-    cat $table | grep '^include ' | sed "s|^include  *\(.[^ ]*\).*$|$(dirname $table)/\1|g"
-done | sort | uniq | \
+ret=0
 while read -r table; do
     if has_metadata $table; then
         echo "Table $table has metadata and is also included somewhere" >&2
-        exit 1
+        ret=1
     fi
-done
+done <<< $(
+    for table in $(dirname $0)/../tables/*; do
+        cat "$table" | dos2unix | grep '^include ' | sed "s|^include  *\([^ ]*\).*$|$(dirname $table)/\1|g"
```

Comment:
Not everybody has `dos2unix` installed. Do we really need it?

Reply:
For some reason the `sed` acted weird for some tables which turned out to be DOS encoded. Because I didn't know how to fix it I added the `dos2unix`. Anyway, since I disabled this test, and the file is only kept for the purpose of possible enabling it again later, I don't think it really matters that much right now. Maybe just add a small comment?

Reply:
I guess we should go through the tables and make sure they're all utf8 just like @mhameed did once. That would be better.

Reply:
They are UTF8, some just have Windows line endings.